src/nti/site/hostpolicy.py

```diff
@@ -123,6 +125,10 @@ def synchronize_host_policies():
                 site = HostPolicyFolder()
                 # should fire object created event
                 sites[name] = site
+                # Some comps are marked a certain way, we want to apply this
+                # to the site itself.
+                if IClientSite.providedBy(comps):
+                    interface.alsoProvides(site, IClientSite)
```

Comment:
This probably needs to happen before the site is stored so that subscribers see this.

Reply:
Good point